src/components/SubTypeAccordionsContainer/index.js

```diff
@@ -1,52 +1,63 @@
 import React from 'react';
 import {
-  StyleSheet, View,
+  StyleSheet, View, Text,
 } from 'react-native';
 import { shape, arrayOf, bool } from 'prop-types';
 
 import BaseAccordion from '../Generic/BaseAccordion';
-import BaseText from '../Generic/BaseText';
+import { PLURAL_RESOURCE_TYPES } from '../../constants/resource-types';
 
-const SubTypeAccordionsContainer = ({ data, fromDetailsPanel }) => (
-  <View>
-    { data.map(({ type, label, subTypes }, index) => {
-      const firstGroupStyle = index === 0 ? styles.firstGroupContainer : {};
-      return (
-        <View key={type} style={[styles.groupContainer, firstGroupStyle]}>
-          {fromDetailsPanel
+const SubTypeAccordionsContainer = ({ data, fromDetailsPanel, fromDateAccordion }) => {
+  const typeTextContainerStyles = [styles.typeTextContainer];
+  if (fromDateAccordion) {
+    typeTextContainerStyles.push({ marginLeft: 30 });
+  }
+  return (
+    <View>
+      { data.map(({ type, subTypes }, index) => {
+        const firstGroupStyle = index === 0 ? styles.firstGroupContainer : {};
+        return (
+          <View key={type} style={[styles.groupContainer, firstGroupStyle]}>
+            {fromDetailsPanel
                 && (
-                <View style={styles.typeTextContainer}>
-                  <BaseText variant="title" style={styles.typeText}>
-                    {label}
-                  </BaseText>
+                <View style={typeTextContainerStyles}>
+                  <View style={styles.typeTextPill}>
+                    <Text style={styles.typeText}>
+                      {PLURAL_RESOURCE_TYPES[type]}
```

Comment:
should this remain as `label`? 

`data`appears to be sorted by `label` (at least in some cases?^*) -- so if they've diverged, then the plumbing for `label` should be fixed?

eg: where a selector uses `sortedGroupedRecordsByType`?
![image](https://user-images.githubusercontent.com/3383704/121543509-f8d7ab00-c9d6-11eb-9611-cf590388e4ec.png)

Reply:
We need this plural variant as we have a group of records with x-ResourceType. See how it's being used below. Additionally from drashko is that these "pill" mimic the `ResourceTypePicker` buttons.

![image](https://user-images.githubusercontent.com/45667486/121546747-887e5900-c9d9-11eb-9776-8464c6ef6565.png)